server/viame_server/event.py

```diff
@@ -0,0 +1,29 @@
+from girder.models.folder import Folder
+from girder.models.item import Item
+
+
+def check_existing_annotations(event):
+    info = event.info
+
+    if "annotations.csv" in info["importPath"]:
+        item = Item().findOne({"_id": info["id"]})
+        item["meta"].update({"folderId": str(item["folderId"]), "pipeline": None})
+        Item().save(item)
+
+        folder = Folder().findOne({"_id": item["folderId"]})
+
+        # FPS is hardcoded for now
+        folder["meta"].update({"type": "image-sequence", "viame": True, "fps": 30})
+        Folder().save(folder)
+
+
+def mark_folder_for_annotation(event):
```

Comment:
This function runs for all uploads, but only has an effect for folders.  You could rename this function `maybe_mark_folder_for_annotation(event)` to indicate that it doesn't always do that.

I think this function is straightforward, so you don't have to, just wanted to point out a pattern I've seen elsewhere in the girder ecosystem (particularly in worker)